packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1027,6 +1063,19 @@ class TabBar extends StatefulWidget implements PreferredSizeWidget {
   /// If this property is null, it is interpreted as [BorderRadius.zero].
   final BorderRadius? splashBorderRadius;
 
+  /// Specifies the horizontal alignment of the tabs within a [TabBar].
```

Comment:
Should this mention the special case around isScrollable?

Reply:
Good idea!